lib/control/v1/index.js

```diff
@@ -13,6 +14,10 @@ exports.attach = function (app, storage) {
   app.get('/v1/cubbyhole/:token/:path', require('./cubbyhole')(storage));
   app.get('/v1/credential/:token/:mount/:role', require('./credential')(storage));
 
+  app.use(BodyParser.json());
```

Comment:
I moved this out the route definitions and into server binary where we set up the logging middleware. See https://github.com/rapid7/tokend/pull/62/commits/c32776be940ebf6901b18c13e7a0a8a2b3280802 for those changes.